install/0000_90_cluster-version-operator_02_servicemonitor.yaml

```diff
@@ -56,6 +57,7 @@ spec:
       expr: |
         (time()-cluster_version_operator_update_retrieval_timestamp_seconds) >= 3600 and ignoring(condition, name, reason) cluster_operator_conditions{name="version", condition="RetrievedUpdates", endpoint="metrics", reason!="NoChannel"}
       labels:
+        namespace: openshift-cluster-version
```

Comment:
Checking Telemetry, `CannotRetrieveUpdates` seems to include `namespace` already via `cluster_version_operator_update_retrieval_timestamp_seconds` and `cluster_operator_conditions` scraping.  So I'd rather drop this line from the commit as redundant.

Reply:
Thank you, fixed in https://github.com/openshift/cluster-version-operator/pull/800/commits/d525eb97a2c971b4837210963d86d3b617452891.